std/math.d

```diff
@@ -2449,17 +2461,17 @@ version (IeeeFlagsSupport) @safe @nogc nothrow unittest
         assert(exp(cast(T) 0.0) == 1.0);
 
         // NaN propagation. Doesn't set flags, bcos was already NaN.
-        resetIeeeFlags();
+        version (IeeeFlagsSupport) resetIeeeFlags();
         x = exp(T.nan);
-        f = ieeeFlags;
+        version (IeeeFlagsSupport) f = ieeeFlags;
         assert(isIdentical(abs(x), T.nan));
-        assert(f.flags == 0);
+        version (IeeeFlagsSupport) assert(f.flags == 0);
 
-        resetIeeeFlags();
+        version (IeeeFlagsSupport) resetIeeeFlags();
         x = exp(-T.nan);
-        f = ieeeFlags;
+        version (IeeeFlagsSupport) f = ieeeFlags;
         assert(isIdentical(abs(x), T.nan));
-        assert(f.flags == 0);
+        version (IeeeFlagsSupport) assert(f.flags == 0);
```

Comment:
Just have two version block. I don't think its very nice to read with every other statement versioned.